docs/content/3.providers/imgproxy.md

```
@@ -0,0 +1,123 @@
+
+## Imgproxy `fit` Values
+
+All `fit` values are converted to [`resizing_type`](https://docs.imgproxy.net/usage/processing#resizing-type) option provided by Imgproxy. 
+The `fit` option is kept for backwards compatibility with @nuxt/image. The `contain` value is converted to `fill`.
+
+## Imgproxy default options
+
+## Imgproxy Modifiers
+
+By default, the Imgproxy provider has the following settings for modifiers
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)